msal/tests/dev apps/DesktopTestApp/MainForm.cs

```diff
@@ -81,11 +82,11 @@ public void LogDelegate(LogLevel level, string message, bool containsPii)
         public void RefreshUserList()
         {
             List<IAccount> accounts = _publicClientHandler.PublicClientApplication.GetAccountsAsync().Result.ToList();
-            if (accounts.Count > 0)
-            {
-                accounts.Insert(0, new Account(_publicClientHandler.CurrentUser.HomeAccountId, 
-                    _publicClientHandler.CurrentUser.Username, _publicClientHandler.CurrentUser.Environment));
-            }
+            //if (accounts.Count > 0)
```

Comment:
>//if (accounts.Count > 0) [](start = 12, length = 25)

?

Reply:
Not sure what it was doing before moving to accounts, but now it just duplicates an account. The UI is already bound to accounts variable.